rules/linux/lateral_movement_telnet_network_activity_external.toml

```diff
@@ -34,7 +34,8 @@ query = '''
 sequence by process.entity_id
   [process where process.name == "telnet" and event.type == "start"]
   [network where process.name == "telnet" and
-    destination.ip not in ("127.0.0.0/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16", "FE80::/10", "::1/128")]
+    not cidrMatch(destination.ip, "127.0.0.0/8", "10.0.0.0/8", "172.16.0.0/12",
```

Comment:
so we have a few options here
* add ipv6 to cidrMatch for EQL. this is a big ask unfortunately
* use `cidrMatch` but monkey patch the existing EQL function, just to pass unit tests
* rename to `cidrmatch` and use `with eql.parser.ignore_unknown_functions:` when parsing

Reply:
lets go with option 3 for this and aim for option 1 for 7.11

Updating now